generic_tracers/generic_COBALT.F90

```diff
@@ -1196,78 +1196,206 @@ subroutine user_add_params(param_file)
-    call get_param(param_file, "generic_COBALT", "phi_srdon_mdz", zoo(2)%phi_srdon,  "phi_srdon_mdz", units="unitless", default= 0.075*0.15)    ! dimensionless
-    call get_param(param_file, "generic_COBALT", "phi_srdon_lgz", zoo(3)%phi_srdon,  "phi_srdon_lgz", units="unitless", default= 0.075*0.0)     ! dimensionless
-    call get_param(param_file, "generic_COBALT", "phi_srdop_smz", zoo(1)%phi_srdop,  "phi_srdop_smz", units="unitless", default= 0.125*0.30)   ! dimensionless
-    call get_param(param_file, "generic_COBALT", "phi_srdop_mdz", zoo(2)%phi_srdop,  "phi_srdop_mdz", units="unitless", default= 0.125*0.15)   ! dimensionless
-    call get_param(param_file, "generic_COBALT", "phi_srdop_lgz", zoo(3)%phi_srdop,  "phi_srdop_lgz", units="unitless", default= 0.125*0.0)    ! dimensionless
-    call get_param(param_file, "generic_COBALT", "phi_sldon_smz", zoo(1)%phi_sldon,  "phi_sldon_smz", units="unitless", default= 0.3*0.30)    ! dimensionless
-    call get_param(param_file, "generic_COBALT", "phi_sldon_mdz", zoo(2)%phi_sldon,  "phi_sldon_mdz", units="unitless", default= 0.3*0.15)    ! dimensionless
-    call get_param(param_file, "generic_COBALT", "phi_sldon_lgz", zoo(3)%phi_sldon,  "phi_sldon_lgz", units="unitless", default= 0.3*0.0)     ! dimensionless
-    call get_param(param_file, "generic_COBALT", "phi_sldop_smz", zoo(1)%phi_sldop,  "phi_sldop_smz", units="unitless", default= 0.3*0.30)    ! dimensionless
-    call get_param(param_file, "generic_COBALT", "phi_sldop_mdz", zoo(2)%phi_sldop,  "phi_sldop_mdz", units="unitless", default= 0.3*0.15)    ! dimensionless
-    call get_param(param_file, "generic_COBALT", "phi_sldop_lgz", zoo(3)%phi_sldop,  "phi_sldop_lgz", units="unitless", default= 0.3*0.0)     ! dimensionless
+    ! References:
+    ! Carlotti et al., 2000. Modeling zooplankton dynamics. Zooplankton Methodology Manual, pp. 571-667.
+    ! Nagata, 2000. Chapter 5 of "Microbial Ecology of the Oceans, 1st ed." (Kirchman ed.)
+    ! Stock and Dunne, 2010. (https://doi.org/10.1016/j.dsr.2009.10.006) 
+    ! Abell et al., 2000. (https://elischolar.library.yale.edu/journal_of_marine_research/2349) 
+    ! Wheeler et al., 1997. (https://doi.org/10.1016/S0967-0637(96)00089-1)
+    ! Vidal et al., 1999. (https://doi.org/10.4319/lo.1999.44.1.0106)
+    !
+    call get_param(param_file, "generic_COBALT", "phi_det_smz", zoo(1)%phi_det, &
+                   "fraction of ingestion by small zooplankton to detritus", units="none", default=0.0)
+    call get_param(param_file, "generic_COBALT", "phi_det_mdz", zoo(2)%phi_det, &
+                   "fraction of ingestion by medium zooplankton to detritus", units="none", default=0.15)
+    call get_param(param_file, "generic_COBALT", "phi_det_lgz", zoo(3)%phi_det, &
+                   "fraction of ingestion by large zooplankton to detritus", units="none", default=0.30)
+    ! partitioning of zooplanton ingestion to labile dissolved organic material
```

Comment:
`zooplankton`

Reply:
Thanks @yichengt900.   I'll have these corrected in a new commit shortly.